guides/common/modules/con_smartproxy-performance-tests.adoc

```diff
@@ -41,5 +41,5 @@ We have tested executing Remote Execution jobs via both SSH and Ansible backend
 All configurations were able to handle all of the tests without errors, except for the smallest configuration (4 CPUs and 12 GiB memory) which failed to finish on all 4000 hosts.
 
 .Content sync use case
-In a sync test where we synced {RHEL} 6, 7, 8 BaseOS and 8 AppStream we have not seen significant differences among {SmartProxy} configurations.
+In a sync test where we synced {RHEL} 7, 8 BaseOS and 8 AppStream we have not seen significant differences among {SmartProxy} configurations.
```

Comment:
I think this is a bit of an edge case. It describes a specific experiment performed, not the supported OS. How relevant that actually is can be debated. IMHO the whole chapter paragraph doesn't say a lot and I'm tempted to drop it.